src/patternfly/components/Button/button.scss

```diff
@@ -64,6 +64,16 @@
   --pf-c-button--m-tertiary--active--after--BorderColor: var(--pf-global--Color--100);
   --pf-c-button--m-tertiary--active--Color: var(--pf-global--Color--100);
 
+  // Warning btn
+  --pf-c-button--m-warning--BackgroundColor: var(--pf-global--warning-color--100);
+  --pf-c-button--m-warning--Color: var(--pf-global--Color--dark-100);
+  --pf-c-button--m-warning--hover--BackgroundColor: var(--pf-global--warning-color--200);
```

Comment:
I checked with @mceledonia and he suggested using `--pf-global--palette--gold-500` for hover/focus/active background color instead, and leaving the text color as it is.

Reply:
Thank you Michael! Sounds good -- I'll make this change.